config/config.go

```diff
@@ -89,13 +90,18 @@ func (c *Config) Update(key, host, dir, xapi string) {
 
 	dir = strings.TrimSpace(dir)
 	if dir != "" {
-		c.SetDir(dir)
+		err = c.SetDir(dir)
+		if err != nil {
```

Comment:
Ditto for this, we don't have to declare the `var err error` above if we scope the error in here.